tests/testTriggers.go

```diff
@@ -508,6 +510,115 @@ func TriggerDeployNewApps(contexts *[]*scheduler.Context, recordChan *chan *Even
 	})
 }
 
+//TriggerVolumeCreatePXRestart create volume , attach , detach and reboot nodes parallely
+func TriggerVolumeCreatePXRestart(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+
+	defer ginkgo.GinkgoRecover()
+	defer endLongevityTest()
+	startLongevityTest(VolumeCreatePxRestart)
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: VolumeCreatePxRestart,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	setMetrics(*event)
+	stepLog := "Create multiple volumes , attached and restart PX"
+	var createdVolIDs map[string]string
+	var err error
+	volCreateCount := 10
+	Step(stepLog, func() {
+		dash.Infof(stepLog)
+
+		stNodes := node.GetStorageNodes()
+		index := randIntn(1, len(stNodes))[0]
+
+		selectedNode := stNodes[index]
+
+		dash.Infof("Creating and attaching %d volumes on node %s", volCreateCount, selectedNode.Name)
+
+		wg := new(sync.WaitGroup)
+		wg.Add(1)
+		go func(appNode node.Node) {
+			createdVolIDs, err = CreateMultiVolumesAndAttach(wg, volCreateCount, selectedNode.Id)
+			if err != nil {
+				UpdateOutcome(event, err)
+			}
+		}(selectedNode)
+		time.Sleep(3 * time.Second)
```

Comment:
same as above

Reply:
this is just to make sure pxrestart in. next step happens after 1 or volumes are created